features/Company/pages/company-table.tsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable react/display-name */
+import React from 'react'
+import MaterialTable from 'material-table'
+import { AddCircle, SettingsApplications } from '@material-ui/icons'
+import Link from 'next/link'
+
+const CompanyTable = () => {
+  const column = [
+    {
+      title: 'Logo',
+      field: 'imageUrl',
+      render: function (rowData) {
+        return (
+          <img alt="logo" src={rowData.imageUrl} style={{ width: '40px', borderRadius: '50%' }} />
+        )
+      }
+    },
+    { title: 'บริษัท', field: 'company' },
```

Comment:
```suggestion
    { title: 'บริษัท', field: 'company_name_th' },
```
อันนี้ต้องเป็น company_name_th หรือป่าว ตาม field ที่หลังบ้านส่งมา